content/learn/05.communication/01.wire/wire.md

```diff
@@ -2,16 +2,34 @@
 title: 'A Guide to Arduino & the I2C Protocol (Two Wire)'
```

Comment:
```suggestion
title: 'Arduino & the I2C Protocol Guide'
```

Reply:
@karlsoderby Part of me thinks the title is too long as it is right now, and that this change would be better for SEO. But I'm not sure, what do you think?

Reply:
Hmm I think we can rename this simply to `Inter-Integrated Circuit (I2C) Protocol`.